etl/steps/data/garden/animal_welfare/2023-08-01/global_hen_inventory.py

```diff
@@ -31,30 +31,65 @@
 }
 
 
+def fix_typos_in_original_metadata(tb: Table) -> Table:
+    tb_fixed = tb.copy()
+
+    # Ensure that certain columns are strings, to avoid issues when modifying a categorical column.
+    tb_fixed = tb_fixed.astype({"source": str, "available_at": str})
+
+    # The url for Uruguay is placed in "source" instead of "available_at".
+    uruguay_mask = tb_fixed["country"] == "Uruguay"
+    expected_text = " Available at: "
+    assert (
+        expected_text in tb_fixed[uruguay_mask]["source"].item()
+    ), f"Expected '{expected_text}' in source for Uruguay."
+    uruguay_url = tb_fixed[uruguay_mask]["source"].item().split(expected_text)[1]
+    tb_fixed.loc[uruguay_mask, "available_at"] = uruguay_url
+    tb_fixed.loc[uruguay_mask, "source"] = tb_fixed.loc[uruguay_mask, "source"].item().split(expected_text)[0]
+
+    # TODO: Malta has no link, and the source seems to be wrong.
+
+    # Fix some typos in the metadata, and ensure sources end in a period.
+    tb_fixed["source"] = (
+        tb_fixed["source"]
+        .str.replace("husbndry", "husbandry")
+        .replace("(Ministry of Agriculture). '", "(Ministry of Agriculture)'")
```

Comment:
maybe you can do `tb_fixed["source"].str.strip()` too, in case this were to happen for other values.

Reply:
Unless this string is something that appears within the text and is not the end of it.

Reply:
This was an odd case, where the text was between quotation marks, strip would not fix this.